### PR TITLE
Commenting out debug print statements and removing unsupported logging package.

### PR DESCRIPTION
**Pull request by QuinnDamerell-MS**

### CODE REVIEW

**Comment by QuinnDamerell-MS** [comment]
Fixes issue #11

File: loggly.go [hunk 1]
```diff
@@ -110,7 +107,7 @@ func (c *Client) Send(msg Message) error {
 
 	c.buffer = append(c.buffer, json)
 
-	debug("buffer (%d/%d) %v", len(c.buffer), c.BufferSize, msg)
+	//debug("buffer (%d/%d) %v", len(c.buffer), c.BufferSize, msg)
```
Comment (stephenmathieson) [inline loggly.go]: Rather than commenting out the `debug()` calls, I'd prefer we just remove them entirely.

File: loggly.go [hunk 2]
```diff
@@ -254,16 +251,16 @@ func (c *Client) Flush() error {
...
 	}
 
 	defer res.Body.Close()
 
-	debug("%d response", res.StatusCode)
+	//debug("%d response", res.StatusCode)
 	if res.StatusCode >= 400 {
-		resp, _ := ioutil.ReadAll(res.Body)
-		debug("error: %s", string(resp))
+		_, _ = ioutil.ReadAll(res.Body)
```
Comment (stephenmathieson) [inline loggly.go]: Please remove the blank identifiers here:

```go
ioutil.ReadAll(res.Body)
```

**Comment by glasser** [comment]
Breaking our build too — looking forward to seeing this merged. Happy to make another version of the PR with lines deleted if that's faster!

File: loggly.go (hunk 2 above)
Comment (stephenmathieson) [reply loggly.go]: Actually, this condition is completely useless and can be removed in full.

**Comment by glasser** [comment]
eg https://github.com/segmentio/go-loggly/pull/13

**Comment by stephenmathieson** [comment]
#13

**Pull request closed without merging**